src/clj/rems/handler.clj

```diff
@@ -136,8 +115,17 @@
 
 (defn app-routes []
   (routes
+   (wrap-login-redirect (routes (GET "/" []
+                                     (layout/home-page))
+                                attachment-routes
+                                redirects))
+   css-routes
+   ;; never cache authentication results
+   ;; TODO this is a slightly hacky place to do this
+   (middleware/wrap-no-cache (auth/auth-routes))
```

Comment:
Perhaps we should use without the ns-prefix here like the other wraps?

Reply:
fine by me

Reply:
(will change in the next PR)